digital_land/cli.py

```diff
@@ -108,17 +108,21 @@ def convert_cmd(input_path, output_path):
 
 @cli.command("dataset-create", short_help="create a dataset from processed resources")
 @click.option("--output-path", type=click.Path(), default=None, help="sqlite3 path")
+@click.option("--max-batch-size", type=click.INT)
```

Comment:
This will break the pipeline, there is no default it could be worth making an acceptance test for creating a dataset